lib/vsc/utils/script_tools.py

```diff
@@ -210,3 +226,178 @@ def critical_exception_handler(self, tp, value, traceback):
         message = "Script failure: %s - %s" % (tp, value)
         sys.exc_clear()
         self.critical(message)
+
+
+class CLI(object):
+    """
+    Base class to implement sync clients.
```

Comment:
CLI means command line interface to me, it seems like a bad name for something about sync clients.